fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricLanguageProvider.java

```diff
@@ -228,6 +229,16 @@ default void add(Identifier identifier, String value) {
 			add(identifier.toTranslationKey(), value);
 		}
 
+		/**
+		 * Adds a translation for an {@link TagKey}.
```

Comment:
```suggestion
		 * Adds a translation for a {@link TagKey}.
```
Missed this previously